packages/devextreme/js/core/utils/license.js

```diff
@@ -0,0 +1,43 @@
+import { parseToken } from '../../__internal/core/license/license_validation';
+import errors from '../errors';
+
+let isLicenseVerified = false;
+
+export function verifyLicense(licenseToken, version) {
+    if(isLicenseVerified) {
+        return;
+    }
+    isLicenseVerified = true;
+
+    let warning;
+
+    if(licenseToken) {
+        const license = parseToken(licenseToken);
+
+        if(license.kind === 'corrupted') {
+            warning = 'W0021';
+        } else {
+            const [major, minor] = version.split('.').map(Number);
+
+            if(major * 10 + minor > license.payload.maxVersionAllowed) {
+                warning = 'W0020';
+            }
+        }
+    } else {
+        warning = 'W0019';
+    }
+
+    if(warning) {
+        errors.log(warning);
+    }
+}
+
+///#DEBUG
```

Comment:
Can we remove this ```//#DEBUG``` section and replace it with something another?
(E.g. mock or something)

We want to get rid of this mechanism in the future (removing some code in debug mode)

Reply:
We decided leave it as is for now